warehouse/integrations/datalake/testdata/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.9"
+
+services:
+  jobsDb:
+    image: postgres:15-alpine
+    environment:
+      - POSTGRES_DB=jobsdb
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_USER=rudder
+    ports:
+      - "5432"
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready" ]
+      interval: 1s
+      retries: 25
+  minio:
+    image: minio/minio:latest
+    ports:
+      - "9000/tcp"
+    environment:
+      - MINIO_ACCESS_KEY=MYACCESSKEY
+      - MINIO_SECRET_KEY=MYSECRETKEY
+      - MINIO_SITE_REGION=us-east-1
+    command: server /data
+    healthcheck:
+      test: curl --fail http://localhost:9000/minio/health/live || exit 1
+      interval: 1s
+      retries: 25
```

Comment:
It is also possible to avoid duplicating common services code by having a common file and per-integration file:

```g
	c := testcompose.New(t, 
	      compose.FilePath("../testdata/common-compose.test.yml"),
	      compose.FilePath("../testdata/local-compose.test.yml"),
	)
```

Reply:
Great point. Let me try this.

Reply:
Added it.